### PR TITLE
Translations for delete account action buttons

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: app/page/template/settings.htm [hunk 1]
```diff
@@ -99,8 +99,8 @@
             <!-- /ko -->
             <!-- ko if: delete_status() === 'dialog' -->
               <p data-uie-name="delete-confirm-text" data-bind="text: delete_confirm_text"></p>
-              <span class="button button-sm" data-uie-name="do-delete-account" data-bind="click: click_on_delete_send">Send</span>
-              <span class="button-label text-theme delete-cancel-button" data-uie-name="do-delete-cancel" data-bind="click: click_on_delete_cancel">Cancel</span>
+              <span class="button button-sm" data-uie-name="do-delete-account" data-bind="click: click_on_delete_send, l10n_text: z.string.extensions_giphy_button_ok"></span>
```
Comment (bennycode) [inline app/page/template/settings.htm]: We need a `z.string.modal_button_send` here

File: .gitignore [hunk 2]
```diff
@@ -25,11 +25,8 @@
 aws-keys.json
 Gruntfile.js
 npm-debug.log
-strings-ar.coffee
-strings-fr.coffee
-strings-it.coffee
-strings-pt.coffee
-strings-ru.coffee
+strings-*.coffee
+!strings-de.coffee
```
Comment (gregor) [inline .gitignore:29]: Me gusta

File: crowdin.yaml [hunk 3]
```diff
@@ -1,3 +1,5 @@
+api_key: <%= ENV['WIRE_CROWDIN_API_KEY'] %>
```
Comment (lipis) [inline crowdin.yaml]: questionable :D

File: trans.py [hunk 4]
```diff
@@ -19,8 +19,11 @@
 
 import os
 
-os.system('crowdin-cli-py upload sources')
-os.system('crowdin-cli-py download')
+home_dir = os.path.expanduser('~')
+user_config = os.path.join(home_dir, 'crowdin.yaml')
```
Comment (lipis) [inline trans.py]: can we do it `.crowdin` for settings

Comment (lipis) [inline trans.py]: never mind.. https://github.com/crowdin/crowdin-cli#split-project-configuration-and-user-credentials :) `crowdin.yaml` it is..

File: trans.py [hunk 5]
```diff
@@ -19,8 +19,11 @@
...
+
+os.system('crowdin-cli-py --identity={} upload sources'.format(user_config))
+os.system('crowdin-cli-py --identity={} download'.format(user_config))
```
Comment (lipis) [inline trans.py:26]: maybe more traditional :)

``` python
os.system('crowdin-cli-py --identity=%s download' % user_config)
```

File: trans.py (hunk 4 above)
Comment (bennycode) [reply trans.py]: But they call it `.crowdin.yaml`. So I will do the same. :)

**Pull request merged**